packages/api/src/command/medical/ihe-result/create-document-query-result.ts

```diff
@@ -0,0 +1,15 @@
+import { DocumentQueryResponseIncoming } from "@metriport/ihe-gateway-sdk";
+import { DocumentQueryResultModel } from "../../../models/medical/document-query-result";
+import { DefaultPayload } from "./shared";
+
+export async function createDocumentQueryResult(
+  defaultPayload: DefaultPayload,
+  status: string,
+  response: DocumentQueryResponseIncoming
+): Promise<void> {
+  await DocumentQueryResultModel.create({
```

Comment:
Usually we return the object/record we just created/updated.

ditto for others